stl/inc/atomic

```diff
@@ -125,6 +125,38 @@ _NODISCARD extern "C" bool __cdecl __std_atomic_has_cmpxchg16b() noexcept;
 #endif
 
 _STD_BEGIN
+// STRUCT TEMPLATE _Storage_for
+struct _Form_mask_t {};
+_INLINE_VAR constexpr _Form_mask_t _Form_mask;
+
+template <class _Ty>
+struct _Storage_for {
+    // uninitialized space to store a _Ty
+    alignas(_Ty) unsigned char _Storage[sizeof(_Ty)];
+
+    _Storage_for()                    = default;
+    _Storage_for(const _Storage_for&) = delete;
+    _Storage_for& operator=(const _Storage_for&) = delete;
+
+    explicit _Storage_for(_Form_mask_t) noexcept {
+        _CSTD memset(_Storage, 0xff, sizeof(_Ty));
+        __builtin_zero_non_value_bits(_Ptr());
+    }
+
+    _Ty& _Ref() noexcept {
+        return reinterpret_cast<_Ty&>(_Storage);
+    }
+
+    _Ty* _Ptr() noexcept {
+        return reinterpret_cast<_Ty*>(_STD addressof(_Storage));
+    }
+};
+
+#if _HAS_CXX20
+template <class _Ty>
+inline constexpr bool _Cmpxchg_has_padding_bits_v =
+    !has_unique_object_representations_v<_Ty> && !is_floating_point_v<_Ty>;

```

Comment:
Should this have a comment about how it won't detect structs with floating-point members, but this is simply for performance, so it's okay for `_Cmpxchg_has_padding_bits_v<FloatingPointPair>` to report `true`?

Reply:
If this is renamed to `_Cmpxchg_might_have_padding_bits` would that resolve your concern without a comment?